site/content/contribute/desktop/developer-setup.md

```diff
@@ -11,6 +11,8 @@ subsection: "Desktop App"
 - Node.js 8 or later
 - NPM
 - Git
+- Windows Build Tools or Visual Studio Community edition (see Known Problems section)
```

Comment:
```suggestion
- Windows Build Tools or Visual Studio Community edition (see Known Problems section).
```
Is the "Known Problems" section in this document? And is it a new section?

Reply:
yes, it is on line 45